packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -26,52 +30,52 @@
           </label>
         </div>
         <div class="user-name">
-        <div class="row">
-          <label class="field-wrapper" for="firstName">
-            First Name
-            <input
-              type="text"
-              id="firstName"
-              maxlength="128"
-              placeholder="First Name"
-              v-model="form.firstName.value"
-              :class="{ error: !form.firstName.isValid }"
-              @blur="validationService.checkFormInput('firstName')"
-            />
-            <ul class="error-color" v-if="form.firstName.errorList.length">
-              <li
-                class="error-color"
-                v-for="error in form.firstName.errorList"
-                v-bind:key="error.length * Math.random()"
-              >
-                {{ error }}
-              </li>
-            </ul>
-          </label>
-        </div>
-        <div class="row">
-          <label class="field-wrapper" for="lastName">
-            Last Name
-            <input
-              type="text"
-              id="lastName"
-              maxlength="128"
-              placeholder="Last Name"
-              v-model="form.lastName.value"
-              :class="{ error: !form.lastName.isValid }"
-              @blur="validationService.checkFormInput('lastName')"
-            />
-            <ul class="error-color" v-if="form.lastName.errorList.length">
-              <li
-                class="error-color"
-                v-for="error in form.lastName.errorList"
-                v-bind:key="error.length * Math.random()"
-              >
-                {{ error }}
-              </li>
-            </ul>
-          </label>
-        </div>
+          <div class="row">
+            <label class="field-wrapper" for="firstName">
+              First Name
+              <input
+                type="text"
+                id="firstName"
+                maxlength="128"
+                placeholder="First Name"
+                v-model="form.firstName.value"
+                :class="{ error: !form.firstName.isValid }"
+                @blur="validationService.checkFormInput('firstName')"
+              />
+              <ul class="error-color" v-if="form.firstName.errorList.length">
+                <li
+                  class="error-color"
+                  v-for="error in form.firstName.errorList"
+                  v-bind:key="error.length * Math.random()"
```

Comment:
I know this is just an indent change but I noticed this-- `v-for` also exposes `index` as a second loop parameter, so we may want to use that:
```js
v-for="(error, index) in form.firstName.errorList"
```

Reply:
If you make that change, also do the same below for the same scenario.

Reply:
and where we want to use index after?

Reply:
Use the index instead of `error.length * Math.random()` in the `key` field